client/pages/configuration/general.vue

```diff
@@ -2,26 +2,38 @@
 const config = useGlobalConfiguration()
 const { t } = useI18n()
 
-const toggleRegistrations = computed(() => config.toggleRegistrations)
-const defaultDiskQuotaPolicy = computed(() => config.defaultDiskQuotaPolicy)
-const defaultDiskQuota = computed(() => config.defaultDiskQuota)
-const maximumThumbnailCount = computed(() => config.maximumThumbnailCount)
+const { cloned: newConfig } = useCloned(config.$state)
+const different = computed(() => JSON.stringify(config.$state) !== JSON.stringify(newConfig.value))
```

Comment:
thing is, JSON.stringify is unstable and a bit slow (not that bad tbh). It is unstable in the sense that if, for some reason, the order in which the properties are declared change, the resulting serialised object will not be the same. 
In our case, this *shouldn't* happen... but I fear it *might*. I leave it up to you to either let things like that, or to implement a proper deep equals function.

something like this:

```ts
function deepEqual(x: any, y: any) {
  if (x === y) {
    return true;
  } else if ((typeof x == "object" && x != null) && (typeof y == "object" && y != null)) {
    if (Object.keys(x).length != Object.keys(y).length) return false;

    for (const prop in x) {
      if (y.hasOwnProperty(prop)) {
        if (!deepEqual(x[prop], y[prop])) return false;
      } else {
        return false;
      }
    }

    return true;
  } else {
    return false;
  }
}
```